src/darwin/Framework/CHIP/zap-generated/MTRBaseClusters.h

```diff
@@ -5129,7 +5496,8 @@ MTR_NEWLY_AVAILABLE
 
 /**
  * Cluster Bridged Device Basic
- *    This Cluster serves two purposes towards a Node communicating with a Bridge: indicate that the functionality on
+ *
+ * This Cluster serves two purposes towards a Node communicating with a Bridge: indicate that the functionality on
```

Comment:
Multiline also seems awkward here and on line 5762 below and probably in more places.
yet multiline at 5891 and others works well. I wonder if that is due to some clang-format or similar. If so, maybe `{description}` should always be one line and let clang format make it nice.

Reply:
This file is not being run through clang-format right now.  But yes, ideally we would address both that and whether the things in the XML make sense.